mkdocs.yml

```diff
@@ -0,0 +1,10 @@
+site_name: approx
```

Comment:
Do we think mkdocs is the way to go? AFAIK, it's far easier to use as compared to sphinx.

Reply:
Yeah, let's use `mkdocs`. I have had my bad memories dealing with `sphinx` 😢

Reply:
Aight